api/video/builtin_video_bitrate_allocator_factory.cc

```diff
@@ -34,6 +34,7 @@ class BuiltinVideoBitrateAllocatorFactory
       case kVideoCodecAV1:
       case kVideoCodecVP9:
         return std::make_unique<SvcRateAllocator>(codec);
+      // TODO: add an allocator here for H.265
```

Comment:
Is this needed or w/o this build process will raise a warning?

Reply:
This comment is picked from old branch.